pages/admin/permit-holders.tsx

```diff
@@ -88,35 +80,42 @@ const COLUMNS = [
   },
   {
     Header: 'Recent APP',
-    accessor: 'recentAPP',
+    accessor: 'recentPermit',
     disableSortBy: true,
     width: 140,
     maxWidth: 140,
-    Cell: renderRecentAPP,
+    Cell: renderRecentPermit,
   },
   {
     Header: 'User Status',
-    accessor: 'userStatus',
+    accessor: 'status',
     disableSortBy: true,
     width: 120,
     maxWidth: 120,
     Cell: renderUserStatusBadge,
   },
 ];
 
+const PAGE_SIZE = 20;
+
 type NameProps = {
   value: {
-    name: string;
-    id: number;
+    firstName: string;
+    middleName: string;
+    lastName: string;
+    rcdUserId: number;
   };
 };
 
+// Name cell in table
 function renderName({ value }: NameProps) {
   return (
     <>
-      <Text>{value.name}</Text>
+      <Text>
+        {value.firstName} {value.middleName} {value.lastName}
```

Comment:
Careful here. `middleName` could be `undefined` and with the way JS works, it will literally render `'undefined'` lol

So we should have something like
```suggestion
        {value.firstName} {value.middleName || ''} {value.lastName}
```